securedrop_client/gui/widgets.py

```diff
@@ -1963,16 +1999,26 @@ def __init__(
         file_missing.connect(self._on_file_missing, type=Qt.QueuedConnection)
 
     def eventFilter(self, obj, event):
-        if event.type() == QEvent.MouseButtonPress:
+        t = event.type()
+        if t == QEvent.MouseButtonPress:
             if event.button() == Qt.LeftButton:
                 self._on_left_click()
+        # HERE BE DRAGONS. Usually I'd wrap this in an if not self.download,
+        # but for reasons not entirely clear, this caused a crash. The
+        # following odd way of expressing the same conditional doesn't cause a
+        # crash. Go figure... :-/
+        if t == QEvent.HoverEnter or t == QEvent.HoverMove and not self.downloading:
```

Comment:
nit: you can remove the check for `HoverMove` it works just fine with `HoverEnter` by itself, unless i'm missing something?